manifests/init.pp

```diff
@@ -120,6 +136,11 @@
     amqp_truststore              => $::certs::candlepin::amqp_truststore,
     qpid_ssl_cert                => $::certs::qpid::client_cert,
     qpid_ssl_key                 => $::certs::qpid::client_key,
+    db_host                      => $katello::candlepin_db_host,
```

Comment:
@sean797 it seems neither the katello class nor the katello::params can see the the db_name's default value from  candlepin. Is there any trick how to do that?